### PR TITLE
Update Catalan translation.

### PR DESCRIPTION
**Pull request by jmallach**
This is a full Catalan translation update for Keycloak 26.

### CODE REVIEW

**Comment by jmallach** [comment]
I kindly request @ECron's review as in #22879.

**Comment by jmallach** [comment]
Sorry, in my initial comment I mentioned Keycloak 25 but in fact the translation was done against master before it was branched to 26, ie the target for this PR is master AND release-26.

**Comment by ssilvert** [comment]
@jmallach We need an issue to be linked to this PR.  Can you create one and link it as per the contribution guidelines?

**Comment by jonkoops** [comment]
@ssilvert we actually do not require any issues to be linked for localization updates up to now. The only requirement is that this PR is reviewed by another native speaker of the language.

**Comment by jmallach** [comment]
@jonkoops I'm waiting for @Ecron to have a look, he just told me via Telegram he'll have a look soonish.

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties [hunk 1]
```diff
@@ -26,16 +26,16 @@ status=Estat
...
 filterByName=Filtra per nom…
 refreshPage=Refresca la pàgina
-error-user-attribute-required=Especifiqueu l''atribut {0}.
+error-user-attribute-required=Especifiqueu l''atribut «{0}».
 currentSession=Sessió actual
-error-empty=Especifiqueu un valor.
-error-invalid-uri-fragment=El fragment d''URL no és vàlid.
-error-invalid-blank=Especifiqueu un valor.
-missingFirstNameMessage=Indiqueu el vostre nom.
+error-empty=Especifiqueu el valor de «{{0}}'.
```
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:35]: Al final cal substituir la coma simple per les cometes llatines (»).

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties [hunk 2]
```diff
@@ -26,16 +26,16 @@ status=Estat
...
+error-invalid-uri-fragment=«{{0}} és un fragment d''URL invàlid.
```
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:36]: Falta una cometa llatina de tancament al final de la variable {{0}}.

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties [hunk 3]
```diff
@@ -64,7 +64,7 @@ logo=Logotip
 password-help-text=Entreu introduint la vostra contrasenya.
 deleteAccount=Elimina el compte
 add=Afegeix
-error-invalid-value=El valor no és vàlid.
+error-invalid-value=«{{0}} té un valor invàlid.
```
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:67]: Potser falta una cometa llatina de tancament al final de la variable {{0}}.

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties [hunk 4]
```diff
@@ -81,14 +81,14 @@ password-display-name=Contrasenya
 expires=Caduca
 noGroups=Cap grup
 notSetUp={0} no està configurat.
-error-invalid-number=El nombre no és vàlid.
+error-invalid-number=«{{0}}» és un nombre invàlid.
```
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:84]: "no válid" millor que "invàlid".

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties [hunk 5]
```diff
@@ -123,3 +123,96 @@ error-person-name-invalid-character=El nom conté un caràcter no vàlid.
...
+avatar=Avatar
+usernamePlaceholder=Nom d'usuari o adreça electrònica
+deny=Denega
+selectALocale=Seleccioneu una llengua
+save=Desa
+resourceSharedWith_one=El recurs està compartit amb <0>{{username}}</0>
+resourceSharedWith_zero=Aquest recurs no està compartit.
+removeConsentError=No s'ha pogut eliminar el consentiment a causa de: {{error}}
+groupsListHeader=Capçalera de llista de grups
+privacyPolicy=Política de privacitat
```
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:141]: Observació: «Privacitat» o «Privadesa»? (qüestió d'estil).

**Comment by ahus1** [comment]
@jmallach / @Ecron - Please keep the discussions in English as much as possible. I know discussing translations might be an edge case, still it allows others to follow the discussion more easily.

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties [hunk 6]
```diff
@@ -123,3 +123,96 @@ error-person-name-invalid-character=El nom conté un caràcter no vàlid.
...
+searchOrganization=Cerca una organització
+organizationList=Llista d'organitzacions
+domains=Dominis
+refresh=Refresca
+scopeNameHelp=Nom de l'abast del client. Ha de ser únic al domini. El nom no ha de contenir caràcters d'espai, ja que s'utilitza com el valor del paràmetre de l'abast
+accountSecurity=Seguretat del compte
+organizationDescription=Mostra les organitzacions a les quals us heu unit
+unLinkError=No s'ha pogut desenllaçar a causa de: {{error}}
+recovery-authn-codes-display-name=Codis d'autenticació de recuperació
+recovery-codes-generate-new-codes=Genereu nous codis per a garantir l'accés al vostre compte
```
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:201]: Potser «codis nous» millor? (qüestió d'estil)

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties [hunk 7]
```diff
@@ -123,3 +123,96 @@ error-person-name-invalid-character=El nom conté un caràcter no vàlid.
...
+clientTypeHelp=El tipus d'aquest recurs. Es pot utilitzar per a agrupar diferents instàncies de recursos del mateix tipus.
+groupsListColumnsNames=Noms de les columnes de llistes de grups
+linkError=No s'ha pogut enllaçar a causa de: {{error}}
+unLinkSuccess=S'ha desenllaçat el compte amb èxit
+webauthn-help-text=Useu la vostra Passkey per a entrar.
+verifiableCredentialsConfigAlert=No s'ha pogut recuperar la configuració de les credencials.
+scopesHelp=Els abasts associats amb aquest recurs.
+createFlowHelp=Podeu crear un flux de primer nivell amb aquest formulari
+scopeTypeHelp=Els abasts del cient, que s'afegiran com a abasts per defecte a cada client creat
+rolesScope=Si no hi ha cap mapatge de l'abast del rol definit, els usuaris tenen permés utilitzar aquest abast de client. Si hi ha definits mapatges de l'abast del rol, l'usuari ha de ser un membre d'almenys un dels rols.
```
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:213]: Millor «permès» si anem amb la varietat central.

File: themes/src/main/resources-community/theme/base/admin/messages/messages_ca.properties [hunk 8]
```diff
@@ -65,3 +61,5 @@ error-invalid-date=L''atribut {0} és una data invàlida.
 error-user-attribute-read-only=L''atribut {0} és només de lectura.
 error-username-invalid-character={0} conté un caràcter no vàlid.
 error-person-name-invalid-character={0} conté un caràcter no vàlid.
+invalidPasswordNotContainsUsernameMessage=La contrasenya no és vàlida: no pot contenir el nom d'usuari.
```
Comment (Ecron) [inline themes/src/main/resources-community/theme/base/admin/messages/messages_ca.properties:64]: Potser s'ha d'escapar l'apòstrof?

File: themes/src/main/resources-community/theme/base/email/messages/messages_ca.properties [hunk 9]
```diff
@@ -44,7 +43,17 @@ linkExpirationFormatter.timePeriodUnit.seconds={0,choice,0#segons|1#segon|1<sego
...
 
+
+orgInviteBodyPersonalized=Hola, {5} {6}.\n\nHeu estat convidats a unir-vos a l''organització «{3}». Feu clic a l''enllaç següent per a unir-vos-hi.\n\n{0}\n\nAquest enllaç caducarà el {4}.\n\nSi no us voleu unir a l''organització, podeu ignorar aquest missatge.
+orgInviteBodyHtml=<p>Heu estat convidats a unir-vos a l'organització «{3}». Feu clic a l'enllaç següent per a unir-vos-hi.</p><p><a href="{0}">Enllaç per a unir-se a l'organització</a></p><p>Aquest enllaç caducarà el {4}.</p><p>Si no us voleu unir a l'organització, podeu ignorar aquest missatge.</p>
+orgInviteBody=Heu estat convidats a unir-vos a l''organització «{3}». Feu clic a l''enllaç següent per a unir-vos-hi.\n\n{0}\n\nAquest enllaç caducarà el {4}.\n\nSi no us voleu unir a l''organització, podeu ignorar aquest missatge.
+orgInviteBodyPersonalizedHtml=<p>Hola, {5} {6}</p><p>Heu estat convidats a unir-vos a l'organització «{3}». Feu clic a l'enllaç següent per a unir-vos-hi.</p><p><a href="{0}">Enllaç per a unir-se a l'organització</a></p><p>Aquest enllaç caducarà el {4}.</p><p>Si no us voleu unir a l'organització, podeu ignorar aquest missatge.</p>
```
Comment (Ecron) [inline themes/src/main/resources-community/theme/base/email/messages/messages_ca.properties:53]: «Heu estat convidats» o «Heu estat convidat»?

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties (hunk 5 above)
Comment (jmallach) [reply js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:141]: I'll switch to privadesa here and in another appearance, but termcat says they are interchangeable.

File: themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties [hunk 10]
```diff
@@ -307,40 +275,38 @@ invalidPasswordNotEmailMessage=La contrasenya no és vàlida: no pot ser igual a
...
 couldNotSendAuthenticationRequestMessage=No s''ha pogut enviar la petició d''identificació al proveïdor d''identitat.
 unexpectedErrorHandlingRequestMessage=Error inesperat durant la petició d''identificació al proveïdor d''identitat.
 invalidAccessCodeMessage=Codi d''accés no vàlid.
-sessionNotActiveMessage=La sessió no està activa
+sessionNotActiveMessage=La sessió no està activa.
 invalidCodeMessage=S''ha produït un error, identifiqueu-vos de nou des de la vostra aplicació.
 cookieNotFoundMessage=No s''ha trobat la galeta. Assegureu-vos que les galetes estan habilitades al vostre navegador.
 insufficientLevelOfAuthentication=El nivell d''autenticació sol·licitat no s''ha satisfet.
-identityProviderUnexpectedErrorMessage=Error no esperat intentant autenticar en el proveïdor d''identitat.
+identityProviderUnexpectedErrorMessage=Error no esperat intentant autenticar amb proveïdor d''identitat
```
Comment (Ecron) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties:309]: Faltaria un article: «... amb el proveïdor...» o «... amb un proveïdor...».

File: themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties [hunk 11]
```diff
@@ -470,24 +431,48 @@ idp-email-verification-display-name=Verificació de correu electrònic
...
+organizationScopeConsentText=Organització
+confirmOverrideIdpContinue=Sí, sobreescriu l'enllaç amb el compte actual
+passkey-login-title=Entrada amb Passkey
+passkey-available-authenticators=Passkeys disponibles
+passkey-unsupported-browser-text=Aquest navegador no implementa Passkey. Proveu-ho amb un altre, o contacteu amb l'administrador.
+passkey-doAuthenticate=Entra amb Passkey
+passkey-createdAt-label=Creada
+passkey-autofill-select=Seleccioneu la vostra passkey
+organization.confirm-membership.title=Esteu a punt d'unir-vos a l'organització ${kc.org.name}
+organization.member.register.title=Creeu un compte per a unir-vos a l''organització ${kc.org.name}
```
Comment (Ecron) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties:462]: Potser no cal escapar l'apòstrof.

File: themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties [hunk 12]
```diff
@@ -470,24 +431,48 @@ idp-email-verification-display-name=Verificació de correu electrònic
...
+organization.select=Seleccioneu una organització per a continuar:
+deleteCredentialTitle=Suprimeix {0}
+invalidPasswordNotContainsUsernameMessage=La contrasenya no és vàlida: no pot contenir el nom d'usuari.
+identityProviderMissingCodeOrErrorMessage=Manca el paràmetre o codi d'error a la resposta del proveïdor d'identitat.
+hidePassword=Amaga la contrasenya
+identityProviderLogoutFailure=Error de sortida de l'IdP SAML
+emailVerifiedAlreadyMessage=La vostra adreça de correu electrònic ja està verificada.
+requiredAction.webauthn-register=Registre de WebAuthn
+webauthn-registration-init-label-prompt=Inseriu l''etiqueta de la vostra passkey registrada
```
Comment (Ecron) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties:471]: Potser no cal escapar l'apòstrof. A les següents línies passa el mateix, hi ha barreja d'apòstrofs escapats i no escapats.

**Review by Ecron — COMMENTED** [review]
Reviewed. Some minor comments and fixes in some strings.

File: themes/src/main/resources-community/theme/base/admin/messages/messages_ca.properties (hunk 8 above)
Comment (jmallach) [reply themes/src/main/resources-community/theme/base/admin/messages/messages_ca.properties:64]: I don't think so, but will look what other teams did.

**Comment by jmallach** [comment]
@ssilvert @jonkoops from my side, things are ready, I have applied the corrections suggested by @Ecron (not all, some were suggestions). Many thanks for the review!

**Comment by jonkoops** [comment]
Great! Please resolve the conversations that have been processed so we can see the status. @Ecron could you approve the PR if it looks good to you like this?

**Comment by jmallach** [comment]
@jonkoops as my main interest was the 26 release, will this appear in that branch as is, or do I need a new PR with the same content, targetting the release branch that references this one?

**Comment by jonkoops** [comment]
Everything that gets merged into `main` will become part of the v26 release. So once this PR is merged it will end up there.

**Review by Ecron — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
Thanks all!

**Pull request merged**